<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gams::studio::option::EnvVarConfigEditor</class>
 <widget class="QWidget" name="gams::studio::option::EnvVarConfigEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTableView" name="EnvVarConfigTableView"/>
   </item>
  </layout>
  <action name="actionInsert">
   <property name="icon">
    <iconset resource="../../icons/icons.qrc">
     <normaloff>:/solid/plus</normaloff>:/solid/plus</iconset>
   </property>
   <property name="text">
    <string>Insert</string>
   </property>
   <property name="toolTip">
    <string>Insert Environment Variables</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Ins</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../../icons/icons.qrc">
     <normaloff>:/solid/delete-all</normaloff>:/solid/delete-all</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete Environment Variable</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="actionMoveUp">
   <property name="icon">
    <iconset resource="../../icons/icons.qrc">
     <normaloff>:/solid/move-up</normaloff>:/solid/move-up</iconset>
   </property>
   <property name="text">
    <string>MoveUp</string>
   </property>
   <property name="toolTip">
    <string>Move Up</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="actionMoveDown">
   <property name="icon">
    <iconset resource="../../icons/icons.qrc">
     <normaloff>:/solid/move-down</normaloff>:/solid/move-down</iconset>
   </property>
   <property name="text">
    <string>MoveDown</string>
   </property>
   <property name="toolTip">
    <string>Move Down</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="actionResize_Columns_To_Contents">
   <property name="text">
    <string>Resize Columns To Contents</string>
   </property>
   <property name="toolTip">
    <string>Resize Columns To Contents</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="toolTip">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSelect_Current_Row">
   <property name="text">
    <string>Select Current Row</string>
   </property>
   <property name="toolTip">
    <string>Select Current Row</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
